daemon/src/model/cfd.rs

```diff
@@ -277,6 +277,17 @@ impl Cfd {
         Ok(margin)
     }
 
+    pub fn calc_counterparty_margin(&self) -> Result<Amount> {
```

Comment:
Not sure if the `calc` prefix adds much. The fact that this is a function already sort of implies that it is a computed property.